packages/esm-patient-vitals-app/src/vitals/print/print.scss

```diff
@@ -0,0 +1,51 @@
+@use "@carbon/styles/scss/spacing";
+@use '@carbon/styles/scss/colors';
+@use '@carbon/styles/scss/type';
+@import '~@openmrs/esm-styleguide/src/vars';
+
+@media screen {
+    .printpage {
+        display: none;
```

Comment:
Fix the indentation in this file. We use a tab size of `2` by default. If your editor is not set up to do this by default, press CTRL + Shift + P and type `Indent using Tabs` and set the configured tab size to `2`.